### PR TITLE
Weekly meeting minutes, 2018-10-04

### PR DESCRIPTION
**Pull request by graeme-a-stewart**

### CODE REVIEW

**Comment by graeme-a-stewart** [comment]
http://hepsoftware.org being down is a known issue... otherwise URL checks are fine AFAICS.

File: organization/_posts/2018/2018-10-04-coordination.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+(Lots of apologies this week from Michel, Andrea, Tommaso, David, Witek -
+it’s a CMS Week and an ATLAS S&C Week!)
+
+News, general matters
+=====================
+-   European Committee for Future Accelerators
+    ([ECFA](https://ecfa.web.cern.ch/)) have launched a survey on
+    “Recognition of individuals in large collaborations”. Aimed at
+    younger researchers; very relevant that people working on software
+    share their experience.
```
Comment (eduardo-rodrigues) [inline organization/_posts/2018/2018-10-04-coordination.md]: and participate in the survey.

File: organization/_posts/2018/2018-10-04-coordination.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+-   IRIS-HEP [*announcement
+    list*](https://groups.google.com/a/iris-hep.org/forum/#!forum/announcements)
+    was publicised, people are encouraged to sign-up.
+-   The hepix-techwatch-wg was launched and had its [*first
+    meeting*](https://indico.cern.ch/event/759721/) last week. The aim
+    it to organise and formalise technology tracking in a wider group.
```
Comment (eduardo-rodrigues) [inline organization/_posts/2018/2018-10-04-coordination.md]: is to organise

File: organization/_posts/2018/2018-10-04-coordination.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+    Sub-groups to be setup.
+    -   Michel, Graeme and Torre are cross-over members with HSF hats.
+    -   Helge - lots of people on the list, but many are known to be
+        busy. So in particular people who have sufficient time to do
+        real work should join.
+    -   Software area does not to be covered as this is an existing HSF
```
Comment (eduardo-rodrigues) [inline organization/_posts/2018/2018-10-04-coordination.md]: Some words too many at the start of the sentence ...

**Comment by graeme-a-stewart** [comment]
Thanks @eduardo-rodrigues for reading carefully as ever!

**Pull request merged**